File: backend/src/main/kotlin/ch/boosters/backend/data/syncConfig/model/SyncConfigDto.kt
package ch.boosters.backend.data.syncConfig.model

import java.util.UUID

data class SyncConfigDto(
    val id: UUID?,
    val name: String?,
    val events: List<EventDto>,
    val teams: List<TeamDto>,
    val sports: List<UUID>
)

data class TeamDto(val id: String, val sourceId: Int)
data class EventDto(val id: String, val sourceId: Int)

File: frontend/src/services/syncConfig/entities/syncConfigDto.ts

export type SyncConfigRequest = {
    id?: string
    name?: string
    events: EventDto[]
    teams: TeamDto[]
    sports: string[]
}
export type TeamDto = {
    id: string
    sourceId: number
}
export type EventDto = {
    id: string
    sourceId: number
}